Sources/PaymentButtons/PaymentButton.swift

```diff
@@ -36,7 +46,9 @@ public class PaymentButton: UIButton {
         self.size = size
         self.insets = insets
         self.label = label
+        self.analyticsService.sendEvent("paypal-button:initialized", buttonType: fundingSource.rawValue)
```

Comment:
General question: do we want to send a specific event for each `PaymentButton` subclass - so credit, pay later, paypal. Or are we sending the same event name for all buttons?

Reply:
Its the same event name for all buttons but we also pass the `buttonType` which would be one of `PayPal` `Pay Later` or `Credit`

Reply:
🤦 it certainly helps if I could read, sorry just totally missed the second half of this event. Take it or leave it, but is it worth considering `payment-button` as the naming here? Wondering incase we add something like a Venmo button in the future would we reused this class.

Reply:
I like that idea!

Reply:
@vradopp what do you think about updating the events in this PR to "payment-button" vs "paypal-button"?

Reply:
Sounds good to me! Just updated